ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -225,10 +225,12 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
-- as: aws-ipi-disconnected-cco-manual-sts-p2-f28
-  cron: 14 16 4 * *
+- as: aws-ipi-disconnected-cco-manual-sts-efs-p2-f28
+  cron: 2 1 28 * *
   steps:
     cluster_profile: aws-qe
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
```

Comment:
This is a 4.11 job, here we are using `v4.14`, is that right?

Reply:
Hmm, good catch,thanks, it just a fake filed(it must set a value) actually it use the `enable-qe-catalousoure` ref's catalogsource(image tag) testing, anyway I'll update it the same with the job version.